packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/WorklistDataGrid.1/Examples.3/RowSelection.jsx

```diff
@@ -254,13 +254,13 @@ const RowSelection = () => {
         rowHeaderIndex={rowHeaderIndex}
         columnWidth="180px"
         ariaLabel="Worklist Data Grid"
-        hasSelectableRows={hasSelectableRows}
-        onRowSelect={onRowSelect}
-        onRowSelectAll={onRowSelectAll}
-        onClearSelectedRows={clearRowSelection}
-        onDisableSelectableRows={disableSelectableRows}
         onColumnSelect={onColumnSelect}
-        onEnableRowSelection={enableRowSelection}
+        hasSelectableRows={hasSelectableRows} // Prop to turn row selection mode on/off
+        onRowSelect={onRowSelect} // For row selection, consumer must provide a callback that the Worklist DataGrid will call when the user selects one or more rows.
+        onRowSelectAll={onRowSelectAll} // For row selection, consumer must provide a callback that the Worklist DataGrid will call when the user selects all rows.
+        onClearSelectedRows={clearRowSelection} // To clear selected rows, consumer must provide a callback that the Worklist DataGrid will call to clear the selection.
+        onDisableSelectableRows={disableSelectableRows} // Consumer must provide a callback that the Worklist DataGrid will call to turn off the row selection mode.
+        onEnableRowSelection={enableRowSelection} // Consumer must provide a callback that the Worklist DataGrid will call to turn on the row selection mode.
```

Comment:
Just noticed these as well.

```suggestion
        hasSelectableRows={hasSelectableRows} // Prop to turn row selection mode on/off
        onRowSelect={onRowSelect} // For row selection, consumer must provide a callback that the Worklist Data Grid will call when the user selects one or more rows.
        onRowSelectAll={onRowSelectAll} // For row selection, consumer must provide a callback that the Worklist Data Grid will call when the user selects all rows.
        onClearSelectedRows={clearRowSelection} // To clear selected rows, consumer must provide a callback that the Worklist Data Grid will call to clear the selection.
        onDisableSelectableRows={disableSelectableRows} // Consumer must provide a callback that the Worklist Data Grid will call to turn off the row selection mode.
        onEnableRowSelection={enableRowSelection} // Consumer must provide a callback that the Worklist Data Grid will call to turn on the row selection mode.
```